sqlmesh/core/linter/definition.py

```diff
@@ -149,7 +151,8 @@ def __init__(
         violation_msg: str,
         model: Model,
         violation_type: t.Literal["error", "warning"],
+        violation_range: t.Optional["Range"] = None,
```

Comment:
Nit: do we need to use a forward ref here?
```suggestion
        violation_range: t.Optional[Range] = None,
```